src/main/resources/db/migration/V2024_14_11_0949__CIV-15462_update_ga_hwf_fee_paid_applicant.sql

```diff
@@ -0,0 +1,5 @@
+/**
```

Comment:
could you please change the file name to something like V2024_11_14_0949__* (change month and date position))